src/h5web/visualizations/heatmap/HeatmapProvider.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, ReactElement, ReactNode } from 'react';
+import { Dims, AxisOffsets } from './models';
+
+export interface HeatmapProps {
+  dims: Dims;
+  data: number[][];
+  axisOffsets: AxisOffsets;
+}
```

Comment:
This is the shape of the context's value (i.e. what's returned by `useProps()`).